app/scripts/lib/ppom/ppom-util.ts

```diff
@@ -242,7 +242,11 @@ function normalizeSignatureRequest(request: PPOMRequest): PPOMRequest {
     return request;
   }
 
-  const typedDataMessage = parseTypedDataMessage(request.params[1].toString());
+  const dataMessageParam = request.params[1];
+  const typedDataMessage =
+    typeof dataMessageParam === 'object'
```

Comment:
Unit tests?

Reply:
I added a unit test for `ppom-util.ts` and `transaction.utils`.